format/src/main/thrift/carbondata.thrift

```diff
@@ -183,11 +183,23 @@ struct FileFooter{
 }
 
 /**
+* Footer for indexed carbon file
+*/
+struct FileFooterVersion3{
```

Comment:
After adding this new struct for V3 format, following is not required in `FileFooter`
```8: optional list<BlockletInfo3> blocklet_info_list3;```